contrib/src/lib.rs

```diff
@@ -6,13 +6,15 @@
 //! These libraries are always kept in-sync with the core Rocket library. They
 //! provide common, but not fundamental, abstractions to be used by Rocket
 //! applications. In particular, contributor libraries typically export types
-//! that implement the `FromRequest` trait, `Responder` trait, or both.
+//! implementing a combination of the `FromRequest`, `FromParam`, and 
+//! `Responder` traits.
 //!
 //! Each module in this library is held behind a feature flag, with the most
 //! common modules exposed by default. The present feature list is below, with
 //! an asterisk next to the features that are enabled by default:
 //!
 //! * [json*](struct.JSON.html)
+//! * [uuid](struct.UUID.html)
```

Comment:
You don't think order by type of thing is better? It's a little weird to have them split up, but I can certainly do so.